browser/help/topics/v4-browser-hts.md

```diff
@@ -0,0 +1,223 @@
+- `strand`:Gene strand.
+- `start`: Gene genomic start position (position only).
+- `stop`: Gene genomic stop position (position only.- `xstart`: Gene genomic start position (format: chromosomeposition).Note that chrX is encoded as 23, chrY as 24, and chrM as 25.
+- `xstop`: Gene genomic stop position (format: chromosomeposition).
+- `exons`: Array containing exon information for gene.
+  - `feature_type`: Exon type (e.g., CDS).
+  - `start`: Exon genomic start position (position only).
+  - `stop`: Exon genomic stop position (position only).
+  - `xstart`: Exon genomic start position (format: chromosomeposition)
+  - `xstop`: Exon genomic stop position (format: chromosomeposition)
+- `transcripts`: Array containing information about transcripts associated with the gene.
+  - `interval`: Struct representing the start and end positions of transcript.
+  - `transcript_id`: Unique transcript ID.
+  - `transcript_version`: Transcript version.
+  - `gene_id`: Unique gene ID.
+  - `gene_version`: Gene version.
+  - `chrom`: Chromosome in which transcript is located.
+  - `strand`: Transcript strand.
+  - `start`: Transcript genomic start position (position only).
+  - `stop`: Transcript genomic stop position (position only).
+  - `xstart`: Transcript genomic start position (format: chromsomeposition).
+  - `xstop`: Transcript genomic stop position (format: chromsomeposition).
+  - `exons`: Array containing transcript exon information.
+  - `feature_type`: Exon type (e.g., CDS).
+  - `start`: Exon genomic start position (position only).
+  - `stop`: Exon genomic stop position (position only).
+  - `xstart`: Exon genomic start position (format: chromsomeposition).
+  - `xstop`: Exon genomic start position (format: chromsomeposition).
+  - `reference_genome`: Reference genome associated with this transcript.
+  - `refseq_id`: Transcript RefSeq identifier.
+  - `refseq_version`: RefSeq version.
+- `hgnc_id`: GeneHGNC identifier.
+- `symbol`: Gene symbol.
+- `name`: Gene name.
+- `previous_symbols`: Set containing previous gene symbols.
+- `alias_symbols`: Set containing alternate gene symbols.
+- `omim_id`: Gene OMIM ID.
+- `ncbi_id`: Gene NCBI ID.
+- `symbol_upper_case`: All-caps gene symbol.
+- `search_terms`: Set containing search terms associated with gene.
+- `reference_genome`: Reference genome build associated with this gene.
+- `flags`: Set containing gene flags this gene.
+- `canonical_transcript_id`: Canonical transcript ID.
+- `mane_select_transcript`: Struct containing MANE Select transcript information.
+  - `matched_gene_version`: Version of the matched gene.
+  - `ensembl_id`: Transcript Ensembl ID.
+  - `ensembl_version`: Ensembl version.
+  - `refseq_id`: Transcript RefSeq ID.
+  - `refseq_version`: RefSeq version.
+- `preferred_transcript_id`: Transcript shown on the gene page by default. Field containsMANE Select transcript ID if it exists, otherwise contains Ensemblcanonical transcript ID.
```

Comment:
```suggestion
- `preferred_transcript_id`: Transcript shown on the gene page by default. Field contains MANE Select transcript ID if it exists, otherwise contains Ensembl canonical transcript ID.
```